docs/getting-started/github-apps/github-apps-troubleshooting-tips.rst

```diff
@@ -0,0 +1,75 @@
+================================================
+Troubleshooting and Frequently Asked Questions
+================================================
+
+..
+    Need to update with info about checking lambda errors in UI https://github.com/dockstore/dockstore/issues/3530
+
+.. contents::
+   :local:
+   :depth: 2
+
+Why should I migrate my existing workflows to use GitHub Apps and a .dockstore.yml?
+----------------------------------------------------------------------------------------
+Installing our Dockstore GitHub App onto your Github repository or organization will automatically sync your workflow on Dockstore whenever code is pushed to GitHub.
+This means less manual work for workflow developers, and less waiting for content to update.
+
+This requires the addition of a ``/.dockstore.yml`` file to your repository on Github.
+This file contains workflow information such as workflow path, test parameter paths, etc. that Dockstore will use to setup
+the corresponding workflow on Dockstore. It's important to note, that you will need a ``/.dockstore.yml`` file on each branch of your Github
+repository if you want to sync multiple branches (versions) of your workflow.
+
+You can read more about it at :doc:`/getting-started/github-apps/github-apps`.
+
+How does this change my development flow?
+-------------------------------------------
+If you have a ``/.dockstore.yml`` file in your master or develop branches on GitHub, any new branches you create from these as your template
```

Comment:
I get what you getting at, that it's a recommended flow. So this section is fine. But edge-case wise, I feel I have tosay, it ultimately comes down to whether a branch has a .dockstore.yml or not, not where it branched from.

So maybe this is fine. It does bring up a thought though, that if you don't want a version synced, you should remove the .dockstore.yml.

Totally fine if you ignore this whole comment.

Reply:
This section was pre-existing and I thought it was helpful to keep, particularly to new developers. I'll reword it a bit more to make it clear. Basically the point is to teach that adding a .dockstore.yml to any branch that you plan to use to create new branches, commonly master and develop, will make it so that version will automatically be added to and synced on Dockstore.

Good point. I'll add that if you don't want a version added then the .dockstore.yml should be removed.